src/adaptation/Makefile.am

```diff
@@ -56,4 +56,3 @@ if ENABLE_ICAP_CLIENT
 SUBDIRS += icap
 libadaptation_la_LIBADD += icap/libicap.la
 endif
-
```

Comment:
Please undo out-of-scope formatting changes like this one.

Reply:
I think I chased them all